### PR TITLE
Photo shoot view setup km

### LINKED ISSUE #44: On the day of the wedding, there should be an option for user to text all guests which photos they should be present for
- [ ] function to compare current date to wedding date
- [ ] conditional rendering if current date === wedding date
- [ ] function to send messages - Twilio?
- [ ] security issues to consider - app that mass messages - like an "are you sure" button/alert

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Adds PhotoShootView component
- Adds button to weddings occurring on current day to initialize photo shoot
- Adds page to set up text notifications
- Adds photo card carousel for use during photo shoot

## Any background context you want to provide? 
- Had a few merge conflicts come in due to rebasing. Maybe we should consider changing to merge for the rest of the project/forever.

## What are the relevant tickets? 
* Addresses
* Closes #44

### CODE REVIEW

File: src/components/AddWeddingForm/AddWeddingForm.tsx [hunk 1]
```diff
@@ -22,8 +22,6 @@ const AddWeddingForm: React.FC = () => {
 
   const submitWedding = (event: React.FormEvent) => {
     event.preventDefault();
-    console.log(date);
```
Comment (ericberglund117) [inline src/components/AddWeddingForm/AddWeddingForm.tsx:25]: Nice catch. Get that console.log out of here

File: src/components/PhotoShootView/PhotoShootView.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  font-family: 'Roboto', sans-serif;
+  text-align: left;
+  font-size: .9em;
+}
+.photoShootCard {
+  color: #221e22;
+  opacity: 80%;
+  font-family: 'Roboto', sans-serif;
+  text-align: left;
+  font-size: 2em;
```
Comment (ericberglund117) [inline src/components/PhotoShootView/PhotoShootView.css:35]: This looks very clean!

File: src/components/PhotoShootView/PhotoShootView.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    event.preventDefault();
+		guests.forEach((guest: any) => {
+			const photos = guest.photos.toString().split(",").join(", ")
+			const message = `Hello ${guest.name}, Thank you for attending the ${name} wedding.
+			The lovely couple would like you to meet them for photos ${time} ${location}
+			You will be participating in photo numbers ${photos}`
+			const phoneNumber = `+1${guest.phone}`
+			// POST request to Twilio to send message
+		})
+    clearInputs();
```
Comment (hannakim91) [inline src/components/PhotoShootView/PhotoShootView.tsx:39]: This is a neat additional piece functionality for the day of the wedding

File: src/components/PhotoShootView/PhotoShootView.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  const sendNotifications = (event: React.FormEvent) => {
+    event.preventDefault();
+		guests.forEach((guest: any) => {
+			const photos = guest.photos.toString().split(",").join(", ")
+			const message = `Hello ${guest.name}, Thank you for attending the ${name} wedding.
+			The lovely couple would like you to meet them for photos ${time} ${location}
+			You will be participating in photo numbers ${photos}`
+			const phoneNumber = `+1${guest.phone}`
+			// POST request to Twilio to send message
+		})
```
Comment (ericberglund117) [inline src/components/PhotoShootView/PhotoShootView.tsx:38]: I'm excited to get Twilio integrated into this!

File: src/components/VendorDashboard/VendorDashboard.tsx [hunk 5]
```diff
@@ -61,8 +63,13 @@ function VendorDashboard() {
         }
         break;
       case 3:
-        const currentWeddings = weddings.filter((a: any) => a.date == Date.now())
-        if(currentWeddings.length) {
+        const currentWeddings = weddings.filter((a: any) => {
```
Comment (hannakim91) [inline src/components/VendorDashboard/VendorDashboard.tsx:66]: It seems very handy to be able to sort the weddings by date! Glad you found a library to help with the dates

**Review by ericberglund117 — APPROVED** [review]

**Review by hannakim91 — APPROVED** [review]
Thanks for the new features and especially figuring out the rebase conflicts!

**Pull request merged**